processing/src/main/java/org/apache/druid/query/topn/TopNQuery.java

```diff
@@ -60,6 +61,7 @@ public class TopNQuery extends BaseQuery<Result<TopNResultValue>>
   private final DimFilter dimFilter;
   private final List<AggregatorFactory> aggregatorSpecs;
   private final List<PostAggregator> postAggregatorSpecs;
+  private TopNAggregatorResourceHelper aggregatorHelper;
```

Comment:
I see it is getting initialized in the constructor. So it can be made final as well, no?